### PR TITLE
update docker setup

### PR DESCRIPTION
**Pull request by trowik**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,20 @@
+FROM python:2.7
```
[the remark this answers is not in the document]
Comment (trowik) [reply Dockerfile]: yes, will do with the django update

File: wait-for-it.sh [hunk 2]
```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env bash
```
[the remark this answers is not in the document]
Comment (trowik) [reply wait-for-it.sh]: ok, will do

File: .dockerignore [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+**/*.jfm
+**/azds.yaml
+**/bin
+**/charts
+**/docker-compose*
+**/Dockerfile*
+**/node_modules
+**/npm-debug.log
+**/obj
+**/secrets.dev.yaml
```
[the remark this answers is not in the document]
Comment (trowik) [reply .dockerignore]: ok

**Comment by winged** [comment]
Much better :)

**Pull request merged**